kreuzberg/_extractors/_pandoc.py

```diff
@@ -570,6 +585,59 @@ def _extract_file_sync(self, path: Path) -> str:
             with contextlib.suppress(OSError):
                 Path(output_path).unlink()
 
+    async def _extract_images_with_pandoc(self, file_path: str) -> list[ExtractedImage]:
+        images = []
+
+        with tempfile.TemporaryDirectory() as temp_dir:
+            media_dir = Path(temp_dir) / "media"
+            media_dir.mkdir()
+
+            try:
+                cmd = [
+                    "pandoc",
+                    str(file_path),
+                    "--extract-media",
+                    str(media_dir),
+                    "-t",
+                    "markdown",
+                    "-o",
+                    "/dev/null",
+                ]
+
+                await run_process(cmd)
+
+                if media_dir.exists():
+                    for img_path in media_dir.rglob("*"):
+                        if img_path.is_file() and img_path.suffix.lower() in {
```

Comment:
Use our shared constants from mime types, or place these as a constant at the top of this file